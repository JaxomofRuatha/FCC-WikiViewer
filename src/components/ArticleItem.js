import React from 'react';

const ArticleItem = props => (
  <div>
    <span>{props.articles.id}</span>
    <span>{props.articles.title}</span>
    <span>{props.articles.extract}</span>
    <img src={props.articles.thumbnail} />
  </div>
);

export default ArticleItem;
