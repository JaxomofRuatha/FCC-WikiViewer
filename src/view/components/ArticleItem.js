import React from 'react';

const ArticleItem = props => (
  <span>Dis is a Wikipedia article.</span>
);

export default ArticleItem;
