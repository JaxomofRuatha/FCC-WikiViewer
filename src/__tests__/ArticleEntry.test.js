import React from 'react';
import renderer from 'react-test-renderer';

import ArticleItem from '../components/ArticleEntry';

describe('Correctly renders an ArticleItem component', () => {
  it('ArticleItem renders without crashing', () => {
    const component = renderer.create(<ArticleItem />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
