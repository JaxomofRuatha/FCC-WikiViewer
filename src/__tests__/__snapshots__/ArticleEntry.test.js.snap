// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly renders an ArticleItem component ArticleItem renders without crashing 1`] = `
<div>
  [object Window]
</div>
`;
